app/client/components/cancel/cancellationSummary.tsx

```diff
@@ -50,5 +50,7 @@ export const CancellationSummary = (cancelType: string) => (
   isCancelled(subscription) ? (
     actuallyCancelled(cancelType, subscription)
   ) : (
-    <GenericErrorScreen loggingMessage="Not actually cancelled" />
+    <GenericErrorScreen
+      loggingMessage={"Not actually cancelled : " + cancelType}
```

Comment:
Again, not a new thing, but we could add a bit more context to this logging to help newcomers?